manifests/d/DATEV/SicherheitspaketCompact/7.8.108.25048/DATEV.SicherheitspaketCompact.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.4.6 $debug=NVS1.CRLF.7-5-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: DATEV.SicherheitspaketCompact
+PackageVersion: 7.8.108.25048
```

Comment:
```suggestion
PackageVersion: 7.8.108.25048
MinimumOSVersion: 10.0.22621.0
```

Reply:
I just inspected the installer codes. It seems to be blocked by the x86 architecture, rather than the OS version or OS edition.